src/script/script.cpp

```diff
@@ -169,7 +169,7 @@ unsigned int CScript::GetSigOpCount(bool fAccurate) const
         if (!GetOp(pc, opcode))
             break;
         if (opcode == OP_CHECKSIG || opcode == OP_CHECKSIGVERIFY ||
-            opcode == OP_CHECKSIGFROMSTACK || opcode == OP_CHECKSIGFROMSTACKVERIFY)
```

Comment:
I think this reflects a bug in an earlier version of CSFS - since it can't be executed except in tapscript, CSFS should not be reflected here (as there's no SigOpCount in tapscript)